plugins/webgui/public/translate/en-US.js

```diff
@@ -2,39 +2,39 @@ module.exports = {
   '首页': 'Home',
   '登录': 'Sign in',
   '注册': 'Sign up',
-  '邮箱': 'Email',
+  '邮箱': 'E-mail',
   '密码': 'Password',
   '找回密码': 'Reset password',
   '验证码': 'Verification code',
   '发送验证码': 'Get code',
   '请选择语言：': 'Choose language:',
-  '快速搭建':'Build fast',
-  '易于配置':'Config easily',
+  '快速搭建':'Fast deployment',
+  '易于配置':'Easy configuration',
   '官方标准':'Official API',
-  '仅依赖Node.js，无需安装数据库（可选MySQL）': 'Just depend on Node.js and SQLite (MySQL optionally)',
-  '带有插件系统，仅需修改配置文件即可运行': 'Based on plugins, just run it after editing the config file',
-  '支持libev和python版本的标准manager API': 'Support the standard manager API of both ss-libev and ss-python',
-  '请输入邮箱地址再点击“找回密码”': 'Please fill in your email address first',
+  '仅依赖Node.js，无需安装数据库（可选MySQL）': 'Depends on Node.js and SQLite (optionally MySQL)',
+  '带有插件系统，仅需修改配置文件即可运行': 'Plugin-based, super-easy deployment',
+  '支持libev和python版本的标准manager API': 'Support standard manager API of both ss-libev and ss-python',
+  '请输入邮箱地址再点击“找回密码”': 'Please fill in your E-mail address first',
   '邮箱不能为空': 'Email address cannot be empty',
   '验证码不能为空': 'Code cannot be empty',
   '密码不能为空': 'Password cannot be empty',
-  '验证码已发至邮箱': 'A verification code has been sent to your email',
-  '用户注册成功':'Your registration is done',
+  '验证码已发至邮箱': 'A verification code has been sent to your E-mail account',
+  '用户注册成功':'Successfully registered.',
   '确定': 'OK',
   '取消': 'Cancel',
   '关闭': 'Close',
   '下一步': 'Next',
 
-  '服务器': 'Server',
-  '用户': 'User',
-  '账号': 'Account',
-  '订单': 'Order',
+  '服务器': 'Servers',
```

Comment:
@t123yh I've seen the screenshots posted below by @gyteng , I go for plural forms now.